oak-store-spi/pom.xml

```diff
@@ -26,7 +26,11 @@
             <instructions>
               <Export-Package>
                 org.apache.jackrabbit.oak.jcr,
-                org.apache.jackrabbit.oak.jcr.observation.filter
+                org.apache.jackrabbit.oak.jcr.observation.filter,
+                org.apache.jackrabbit.oak.json,
+                org.apache.jackrabbit.oak.spi.cluster,
+                org.apache.jackrabbit.oak.spi.commit,
+                org.apache.jackrabbit.oak.spi.state
```

Comment:
good point. I added to the existing setup, but I guess it's worthwhile to make it a * export. (I think we had that in oak at some point, but it got changed due to accidental package exports maybe?).